stories/locfeature.HYPER.mdx

```diff
@@ -25,8 +25,12 @@ taxonomy:
     values:
       - Hyperwall
 ---
-
-import Carousel from './components/carousel.jsx';
+import Carousel from "../overrides/common/embedded-video-carousel";
```

Comment:
@agurvich I hope how to use this carousel is clear to you. I used the `carousel_content.json` to feed the data to carousel.